### PR TITLE
a few updates

### PR DESCRIPTION
**Pull request by nelsonic**
Removed almost half of start-here.
(*I wrote most of it, and it was stale...*)

Please feel review/merge. We can add stuff back in as required or make further reductions later. thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
-And we are making ***Everything Open Source*** along the way,
-so that you too can **D**o **W**hat **Y**ou **L**ove!
+We have *all* felt the *frustration* of using products that had/have flaws
+(*and been unable to fix them*) <br/>
+The aim of dwyl is to address *both* of these "*bugs*" *symultaneously*.
 
-**We simplify. We enable people to get through their _need-tos_ efficiently so they can get to their _want-tos_.**
+If you'd like to know more, *please read* ***our [manifesto](/manifesto.md)*** ... the short version is:
+"***treat others as one would like others to treat oneself***" ~
+[**The Golden Rule**]()
```
Comment (iteles) [inline README.md]: Looks like it's missing the link URL here

**Comment by iteles** [comment]
@nelsonic I've reviewed and made a few very minor changes, finishing a couple of incomplete sentences and moving some things around.    
Most of the insertions are simple updates to the markdown as I was going through.

Could you have a quick glance through before I merge to make sure you don't disagree with any changes?

Thanks.

**Comment by nelsonic** [comment]
Underscores work for *italicising* text *most* of the time.
But given that GitHub's markdown parser does not ***always*** parse them correctly, I've stopped using them. See: https://github.com/dwyl/start-here/tree/8bbd28d2ab0c3b5a2a266a1e41fd160fc6ee3038#web-development-_fun_damentals #**fail**

**Comment by nelsonic** [comment]
Unless the formatting is *geniuinly* "broken" (i.e. *not rendering on* ***GitHub***), *please* don't waste your time changing it. **75 additions and 76 deletions** is quite time consuming for *me* to review and does not add as much *value* as simply adding in-line comments pointing out the "*incomplete sentences*"

**Comment by iteles** [comment]
@nelsonic Agreed, I started doing it unconsciously. Won't be doing it again for sure.

**Comment by iteles** [comment]
Commit d85521e added to resolve the issue below where beginning of the list was on the same line as the description and therefore not actually ordering the list.
screen shot 2015-09-17 at 13 18 43

**Pull request merged**